.idea/sbt.xml

```diff
@@ -15,26 +15,38 @@
             <option value="$PROJECT_DIR$/scala/compiler-shared" />
             <option value="$PROJECT_DIR$/scala/conversion" />
             <option value="$PROJECT_DIR$/scala/decompiler" />
+            <option value="$PROJECT_DIR$/scala/dfa" />
             <option value="$PROJECT_DIR$/scala/integration/android" />
             <option value="$PROJECT_DIR$/scala/integration/copyright" />
             <option value="$PROJECT_DIR$/scala/integration/devKit" />
             <option value="$PROJECT_DIR$/scala/integration/gradle" />
-            <option value="$PROJECT_DIR$/scala/integration/intelliLang" />
+            <option value="$PROJECT_DIR$/scala/integration/intellilang" />
             <option value="$PROJECT_DIR$/scala/integration/java-decompiler" />
             <option value="$PROJECT_DIR$/scala/integration/maven" />
+            <option value="$PROJECT_DIR$/scala/integration/ml-completion" />
+            <option value="$PROJECT_DIR$/scala/integration/packagesearch" />
             <option value="$PROJECT_DIR$/scala/integration/properties" />
             <option value="$PROJECT_DIR$/scala/macros" />
             <option value="$PROJECT_DIR$/scala/nailgun" />
             <option value="$PROJECT_DIR$/scala/runners" />
             <option value="$PROJECT_DIR$/scala/scala-api" />
             <option value="$PROJECT_DIR$/scala/scala-impl" />
+            <option value="$PROJECT_DIR$/scala/test-integration/scalatest-finders" />
+            <option value="$PROJECT_DIR$/scala/test-integration/scalatest-finders/tests-2" />
+            <option value="$PROJECT_DIR$/scala/test-integration/scalatest-finders/tests-3_0" />
+            <option value="$PROJECT_DIR$/scala/test-integration/scalatest-finders/tests-3_2" />
+            <option value="$PROJECT_DIR$/scala/testRunners" />
+            <option value="$PROJECT_DIR$/scala/testRunners_spec2_2x" />
+            <option value="$PROJECT_DIR$/scala/traceLogViewer" />
+            <option value="$PROJECT_DIR$/scala/traceLogger" />
             <option value="$PROJECT_DIR$/scala/uast" />
             <option value="$PROJECT_DIR$/scala/worksheet" />
-            <option value="$PROJECT_DIR$/target/tools/idea-runner" />
+            <option value="$PROJECT_DIR$/scala/worksheet-repl-interface" />
             <option value="$PROJECT_DIR$/target/tools/zinc" />
+            <option value="$PROJECT_DIR$/tasty/runtime" />
           </set>
         </option>
-        <option name="sbtVersion" value="0.13.18" />
+        <option name="sbtVersion" value="1.4.9" />
```

Comment:
did you change to this sbt version locally or how did this version come in?

Reply:
I didn't mean to check in any changes to this file. I guess it auto-updated when I re-imported the SBT project, maybe it is out of date? I'll remove it from this PR in any case.